common.mk

```diff
@@ -2,50 +2,34 @@
 
 BIN := $(GDK)/bin
 LIB := $(GDK)/lib
+export PATH := $(BIN):$(PATH)
```

Comment:
I would rather avoid to modify the _PATH_ variable as it may cause troubles for an user who already has a GCC toolchain in its path (it was the case before and we got many issues with that). The way SGDK is working currently is convenient as it doesn't need any environment vars nor _PATH_ modification. The `$(BIN)` var contains full SGDK bin path and the `-B$(BIN)` option allow to set the path for GCC binaries, libs... so it works out of the box :)